docs/index.asciidoc

```diff
@@ -37,6 +37,19 @@ can be https://dev.maxmind.com/geoip/geoip2/geolite2[downloaded from here].
 
 If you would like to get Autonomous System Number(ASN) information, you can use the GeoLite2-ASN database.
 
+[id="plugins-{type}s-{plugin}-database_license"]
+==== Database License
+
+MaxMind changed from releasing the database under a CC license to a proprietary EULA,
+the EULA requires Logstash to update the MaxMind database for our users within 30 days of MaxMind updating their database.
+
+For Logstash default distribution user, the plugin manages database for you.
+With the default setting, an auto-update makes sure database run in the latest version.
+For any reason Logstash fails to download database for 30 days, Logstash will stop the plugin pipeline in order to be compliant.
+If user config `database` option to a path, the plugin does not check for database update, and thus user has to be compliant on their own.
+
+For Logstash open source user, the plugin use the Maxmind CC license database by default.
```

Comment:
```suggestion
https://www.maxmind.com[MaxMind] changed from releasing the GeoIP database under
a Creative Commons (CC) license to a proprietary end-user license agreement
(EULA).  The MaxMind EULA requires Logstash to update the MaxMind database
within 30 days of a database update. If Logstash fails to download the database
for 30 days, it will stop the pipeline in order to maintain compliance.

The GeoIP filter plugin can manage the database for users running the Logstash default
distribution, or you can manage
database updates on your own. The behavior is controlled by the `database`
setting. 
When you use the default `database` setting, the auto-update feature ensures that the plugin is
using the latest version of the database.
Otherwise, you are responsible for maintaining compliance.

The Logstash open source distribution uses the MaxMind Creative Commons license
database by default.
```

Reply:
Overall: There are some instances of "Maxmind" in some of the older sections. Will you please change them all to  to "MaxMind" for accuracy?

Reply:
👌